static/templyte/src/app/tutorial/tutorial.component.ts

```diff
@@ -0,0 +1,57 @@
+import {AuthService} from '../providers/auth.service';
+import {Router} from '@angular/router';
+import {User} from 'firebase';
+import {ApiInterfaceService} from '../providers/api-interface.service';
+import {Component, OnInit} from '@angular/core';
+
+import {UploadService, Upload} from '../upload/upload.service';
+import {AngularFireDatabase} from 'angularfire2/database';
+import {MatDialog} from '@angular/material/dialog';
+import {InputValidateDialogComponent} from '../input-validate-dialog/input-validate-dialog.component';
+import {UploadSuccessDialogComponent} from '../upload-success-dialog/upload-success-dialog.component';
```

Comment:
Unused imports.

Reply:
Maintained the necessary imports, removed the others